template/bsconfig.json

```diff
@@ -1,30 +1,21 @@
 {
   "name": "my-rescript-app",
   "reason": {
-    "react-jsx": 3
+      "react-jsx": 3
```

Comment:
tab sizes in json files have been changed unnecessarily. (`package.json` and `bsconfig.json`)